django-check/action.yaml

```diff
@@ -40,6 +40,12 @@ runs:
           cd ${{ inputs.path }}
           pip install pipenv
           pipenv install --deploy --dev
+      - name: Test (run in parallel)
+        shell: bash
+        run: |-
+          cd ${{ inputs.path }}
+          pipenv run coverage run --concurrency=multiprocessing manage.py test --settings=${{ inputs.projectName }}.settings.ci --parallel
```

Comment:
What's the rationale for moving this first? Usually, it's good practice to run the linter locally on your computer before making a PR change, so we can eliminate the "see if tests fail or not" by just making sure the code is linted beforehand.

The rationale is that if your code fails lint, you should lint it and push before we want to run the tests.

If you want to see, on the event that the test has failed lint, also see if the tests also pass or fail, you can use the `continue-on-error` flag with the linter and make sure you ultimately fail this step.

Reply:
The use case is that (at least in labs) we often lint as a last step, but we need to test incrementally.

It seems like `continue-on-error` is a better approach here. I'll add that and re-request.